StandaloneMmPkg/Library/StandaloneMmCoreEntryPoint/Arm/StandaloneMmCoreEntryPoint.c

```diff
@@ -1232,24 +1134,8 @@ CEntryPoint (
     goto finish;
   }
 
-  // Set the gHobList to point to the HOB list saved in ConfigurationTable[].

```

Comment:
Why was this comment dropped?

Reply:
It adds in another place to explain detail
```
+  // ProcessModuleEntryPointList() copies the HOB List passed
+  // by TF-A, i.e. HobStart, in the ConfigurationTable[].
+  // Therefore, find the HobList in the ConfigurationTable[] by
+  // searching for the gEfiHobListGuid.
+  // Also update the gHobList to point to the HobList in the
+  // ConfigurationTable[] as the HobList passed by TF-A can
+  // be overwritten by StMM after StMM Core is initialised, i.e.
+  // after the MM Core entry point is called.
```